dough-reflection/src/main/java/io/github/thebusybiscuit/dough/reflection/ReflectionUtils.java

```diff
@@ -22,8 +22,17 @@
 
     private ReflectionUtils() {}
 
+    private static int majorVersion;
     private static String currentVersion;
 
+    static {
+        currentVersion =  Bukkit.getServer().getClass().getPackage().getName()
+            .substring(Bukkit.getServer().getClass().getPackage().getName().lastIndexOf('.') + 1);
+        // Turn v1_17_R1 -> 17
+        majorVersion = Integer.parseInt(currentVersion.replace("v1_", "").replace("_R1", ""));
```

Comment:
Why does this deviate?
https://github.com/TheBusyBiscuit/CS-CoreLib2/pull/191/files#diff-3fdffaed6aead00eb6f634b13eebd2f40895ec64443fd02e4ac71b86e5c39581R46-R54

Reply:
Maybe I resolved the merge conflicts incorrectly, let me check again.

Reply:
That is very odd. I guess I'll just add another commit which copy-pastes the current cscl2 ReflectionUtils ontop of the current cherry-pick.